README.md

```diff
@@ -1,13 +1,62 @@
-<p align="center">
-  <a href="https://www.maxmind.com">
-    <img 
-      alt="Gatsby" 
-      src="https://www.maxmind.com/images/maxmind-header-logo.svg" 
-      width="250"
-    />
-  </a>
-</p>
-
 <h2 align="center">
-  Developer Site
-</h2>
\ No newline at end of file
+  <img 
+    alt="MaxMind" 
+    src="src/images/maxmind-logo-with-styles.svg" 
+    width="300"
+  />
+
+  Developer Documentation
+</h2>
+
+---
+
+## Overview
+
+* __Static Site Generator__: GatsbyJS + TypeScript + CSS Modules + MDX
+* __Linting__: ESLint + StyleLint + RemarkLint
+
+
+## Usage
+
+* [Minimum Requirements](#minimum-requirements)
+* [Installation](#installation)
+* [Development](#development)
+  * [Development Server](#development-server)
+  * [Static Server](#static-server)
+* [Testing](#testing)
+* [Deployment](#deployment)
+
+### Minimum Requirements
+* Node 10.13.0
+* Yarn 1.17.3
+
+If you need help installing and/or managing Node and Yarn versions, check out [Volta](https://docs.volta.sh/guide/) (recommended) or [NVM](https://github.com/nvm-sh/nvm).
```

Comment:
I'm not comfortable with recommending Volta while it's still relatively new and its support is lacking relative to the incumbent NVM.